conmon/conmon.c

```diff
@@ -780,6 +819,28 @@ int main(int argc, char *argv[])
 						pexit("Failed to add client socket fd to epoll");
 					}
 					ninfo("Accepted connection");
+				} else if (!exec && evlist[i].data.fd == ctlfd) {
+					num_read = read(ctlfd, buf, BUF_SIZE);
```

Comment:
I have added logic to handle multiple lines and partial lines. PTAL. I will remove the debugs/ninfos before we merge this. Thanks!